src/scripts/active-inactive-orgunit/index.js

```diff
@@ -44,27 +45,19 @@ function OrganisationUnitActivator(conf) {
 
     //this variable is used to control if there are pending asynchronous calls to the server
     this.asyncCalls = 0;
-
     console.log("\nConfig:\n", JSON.stringify(conf, null, "\t"));
 };
 
 /**
  * Run script
  */
 OrganisationUnitActivator.prototype.run = function () {
-    console.log("\nLoading script...");
-    var _this = this;
-    //Process every dataElement group
-    var dataValues;
-    this.dataElementGroups.forEach(function (dataElementGroup) {
-        _this.dataValues = [];
-        console.info("\nFound dataElementGroup..." + dataElementGroup);
-        _this.processDataElementGroup(dataElementGroup);
-    });
-
+    console.log("\nLoading script..."); 
+    //Process the dataElement group 
+    var dataValues = [];  
+    this.processDataElementGroup(this.dataElementGroup, dataValues); 
```

Comment:
Do not pass this.dataElementGroup as a parameter, it doesn't make any sense. If it is inside 'this' you have access to the variable from processDataElementGroup, right?